src/pages/Stream.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import { ZegoUIKitPrebuilt } from "@zegocloud/zego-uikit-prebuilt";
+
+const Stream = () => {
+  const { streamId } = useParams();
+  
+  function randomID(len) {
+		let result = "";
+		if (result) return result;
+		var chars =
+				"12345qwertyuiopasdfgh67890jklmnbvcxzMNBVCZXASDQWERTYHGFUIOLKJP",
```

Comment:
Hi @Jish261203 What is this string here under chars. If it has to be protected then you should keep it in env file?? Explain a bit more about this 2 line.

Reply:
Hi @ashutoshgithubs  the string "12345qwertyuiopasdfgh67890jklmnbvcxzMNBVCZXASDQWERTYHGFUIOLKJP" contains a mix of digits (0-9), lowercase letters (a-z), and uppercase letters (A-Z).This string act as pool of characters from ehich function will randomly selects  to create a unique Id